web/pdf_find_bar.js

```diff
@@ -88,6 +93,7 @@ var PDFFindBar = (function PDFFindBarClosure() {
       event.initCustomEvent('find' + type, true, true, {
         query: this.findField.value,
         caseSensitive: this.caseSensitive.checked,
+        phraseSearch: this.phraseSearch.checked,
```

Comment:
I am not sure that it is needed in scope of this PR. This change will not be correlated with other similar code.